core/src/Kokkos_Macros.hpp

```diff
@@ -522,6 +522,7 @@ static constexpr bool kokkos_omp_on_host() { return false; }
     KOKKOS_IMPL_STRIP_PARENS(CODE)   \
   }
 #else
+#include <openacc.h>
```

Comment:
What compiler did you test with?
(If that is sensitive information you can reach out to me directly)

Reply:
We use Clacc, another OpenACC compiler built on LLVM, which will support both NVIDIA GPUs and AMD GPUs.
(For now, Clacc is still under development, and thus not all Kokkos implementations can be compiled by Clacc.)